dev-docs/development.md

```diff
@@ -176,9 +176,9 @@ All PRs are deployed in a staging enviroment using Heroku. Login using the email
 
 ## Credentials
 
-External contributors should provide credentials via a `.env.development` file [(view example)](.env.development.example).
+External contributors should provide credentials via a `.env.development` file [(view example)](.env.development.example). Note that if you're using the devcontainer (such as with GitHub Codespaces), you must rebuild the container after modifying the `.env.development` file to pull in the new variables.
```

Comment:
```suggestion
External contributors should provide credentials via a `.env.development` file [(view example)](.env.development.example). Developers using the `devcontainer` setup (eg. in GitHub Codespaces), will need to rebuild the container after modifying the `.env.development` file to pull in the new variables.
```
Hmm - this is slightly odd - does `bin/rails s` and rerunning it not also reload the credentials? I might be missing something here.

Reply:
From what I understood the only thing pulling in credentials from .env.development was the docker compose configuration:
https://github.com/hackclub/hcb/blob/6abe345efca463847b7a10bbe3fa52008bbcd8a5/.devcontainer/docker-compose.yml#L20

When I was first setting this up I tried re-running `bin/rails dev` and the environment variables didn't seem to update